src/main/java/core/basesyntax/operation/impl/PurchaseOperationHandler.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.operation.impl;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.operation.OperationHandler;
+
+public class PurchaseOperationHandler implements OperationHandler {
+    private FruitDao fruitDao;
+
+    public PurchaseOperationHandler(FruitDao fruitDao) {
+        this.fruitDao = fruitDao;
+    }
+
+    @Override
+    public void get(FruitTransaction fruit) {
```

Comment:
```suggestion
    public void get(FruitTransaction fruitTransaction) {
```
the same for other handlers